components/Login.jsx

```diff
@@ -1,7 +1,49 @@
 import React from "react";
 
 const Login = () => {
-  return <div>Login</div>;
+  return (
+    <div className="font-bold w-[694px] h-[564px] m-10">
```

Comment:
For the values here, we won't want to hardcode the values, otherwise on mobile it will look weird. Instead we want to use relative units like %. I added a commit which will setup how the component will be called, so you can assume that the login form is now in its proper place and build it responsively from there. It will pop up on the index (home) page of the website!